app/views/admin_mailer/threshold_email_reminder.html.erb

```diff
@@ -1,6 +1,6 @@
 
 <p><%= @petitions.length %> <%= @petitions.length == 1 ? 'petition requires action' : 'petitions require action' %></p>
 
-<p>Please go here <%= link_to nil, threshold_admin_petitions_url %></p>
+<p>Please go here <%= link_to nil, admin_petitions_url %></p>
```

Comment:
I guess this could include a state param to get them to the correct list of petitions by default?